### DISCUSSION TITLE
Make PvP prices great again

### DISCUSSION THREAD

**Discussion by Mikel0ke** [chromiecraft/chromiecraft#7183] [Suggestions]
![pvp prices 2](https://github.com/chromiecraft/chromiecraft/assets/83586527/6d48f918-6cfe-47b1-9039-cd03c75ecfd1)

**Comment by muhanamed** [comment]
wow, very good contribution, I hope to see it live ppc

**Comment by leastretardedccplayer** [comment]
Feeling very Pog about this post, surely it will come through Clueless

**Comment by heyitsbench** [comment]
https://gist.github.com/heyitsbench/a845c1039c27843a18a4a4619d39da7f

**Comment by Never6** [comment]
this will turn 3v3 wsg into 5v5 wsg this is very exciting news

**Comment by Chobungus** [comment]
As a member of the pvp and lgbt community I support this, good work!!

**Comment by Eledor-collab** [comment]
let's goooo!

**Comment by UndoUreche** [comment]
this deserves more love

**Comment by Belodina** [comment]
Let this not be forgotten. Consider it an end of expansion alt-friendly improvement. As well as improving new player experience.
